eng/test-job.yml

```diff
@@ -323,6 +323,26 @@ jobs:
           - forcerelocs
           - gcstress15
 
+    # Zip tests
+    - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+      - script: pushd $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper) && tar -czvf $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).tar.gz ./* && popd
+        displayName: Tar Tests
+    - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
```

Comment:
Do we want `Core_Root` to be part of the artifacts? My preference is not to include the following directories:
* bin
* Common
* Tests
* TestWrappers 

`bin/tests/$(osGroup).$(archType).$(buildConfigUpper)`

Reply:
Please use `_BuildConfig` instead of `buildConfigUpper`. The `_BuildConfig` is what the rest of arcade uses and we should start all agreeing on a standard set of variables.

Reply:
@AaronRobinsonMSFT I would prefer to not use _BuildConfig as it deviates from the rest of the file. If we want to make this change, I believe we should make it all at one time.

Reply:
@echesakovMSFT I would prefer for Core_Root to be published. Ideally one should be able to download the folder and run local tests and repro exactly the CI ran the tests at the time it made them. Specifically this requires the Core_Root that was generated.